### PR TITLE
Update Audittrail to allow Nakadi functionality

### PR DESCRIPTION
**Pull request by zaklawrencea**
This PR updates the Audittrail adapter version. The new version includes functionality which allows the adapter to send events to Nakadi. To enable this, a new config item `audittrail_nakadi_url` has been introduced which will be passed to the adapter to set the target Nakadi URL. In addition, the PlatformCredentialsSet for the adapter has been updated to allow write permission to Nakadi event streams.

### CODE REVIEW

File: cluster/manifests/audittrail-adapter/credentials.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{{- if .Cluster.ConfigItems.audittrail_url }}
+{{- if or (ne .Cluster.ConfigItems.audittrail_url "") (ne .Cluster.ConfigItems.audittrail_nakadi_url "") }}
```
Comment (linki) [inline cluster/manifests/audittrail-adapter/credentials.yaml]: Would that also work?
```
{{- if or .Cluster.ConfigItems.audittrail_url .Cluster.ConfigItems.audittrail_nakadi_url }}
```
Looking at the old version, empty string seems to be `false`.

Comment (zaklawrencea) [reply cluster/manifests/audittrail-adapter/credentials.yaml]: Yes, that should also work (not 100% sure if the parentheses are still required for the `or` though, but I don't think so without the strict comparison).

**Comment by szuecs** [comment]
if it's ready please drop the do-not-merge label, but I always check all PRs that have not the label :)

File: cluster/manifests/audittrail-adapter/credentials.yaml (hunk 1 above)
Comment (zaklawrencea) [reply cluster/manifests/audittrail-adapter/credentials.yaml]: Updated to use the simplified formatting.

**Comment by zaklawrencea** [comment]
:+1:

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**